docs/ref/forms/formsets.txt

```diff
@@ -1,18 +1,78 @@
-=================
-Formset Functions
-=================
+=======
+Formset
+=======
 
 Formset API reference. For introductory material about formsets, see the
 :doc:`/topics/forms/formsets` topic guide.
 
 .. module:: django.forms.formsets
    :synopsis: Django's functions for building formsets.
 
+``Making a formset``
+====================
+
+.. note::
+    From 3.0 and onwards, a formset can be made using two ways either
```

Comment:
This should be a free standing discussion of the two methods, with a `versionadded` (that we can later remove) for the declarative approach.

Reply:
I didn't understand what you mean by free standing discussion